build.gradle

```diff
@@ -39,7 +39,7 @@ def versions = [
         launchDarklySdk    : "5.2.3",
         junit              : '5.8.1',
         junitPlatform      : '1.8.1',
-        log4j              : '2.15.0'
+        log4j              : '2.16.0'
 
```

Comment:
I think this is already `2.16.0` in master

Reply:
merged master with current branch